payments-core/src/main/java/com/stripe/android/payments/PaymentFlowResultProcessor.kt

```diff
@@ -206,7 +207,8 @@ internal sealed class PaymentFlowResultProcessor<T : StripeIntent, out S : Strip
         clientSecret: String,
         requestOptions: ApiRequest.Options
     ): Result<T> {
-        var remainingRetries = originalIntent.paymentMethod?.type?.afterRedirectAction?.retryCount ?: MAX_RETRIES
+        val maxRetries = originalIntent.paymentMethod?.type?.afterRedirectAction?.retryCount ?: MAX_RETRIES
+        var remainingRetries = maxRetries - 1
```

Comment:
Hmm, this is technically a behavior change, which is why I had to update a test to not flake 🤔